common/supplemental/likelySubtags.xml

```diff
@@ -2085,11 +2105,7 @@ not be patched by hand, as any changes made in that fashion may be lost.
 		<likelySubtag from="ria" to="ria_Latn_IN"/>
 		<!--{ Riang (India); ?; ? } => { Riang (India); Latin; India }-->
 		<likelySubtag from="rif" to="rif_Latn_MA"/>
-		<!--{ Riffian; ?; ? } => { Riffian; Latn; Morocco }-->
-		<likelySubtag from="rif_Tfng" to="rif_Tfng_MA"/>
-		<!--{ Riffian; Tifinagh; ? } => { Riffian; Tifinagh; Morocco }-->
-		<likelySubtag from="rif_NL" to="rif_Latn_NL"/>
-		<!--{ Riffian; ?; Netherlands } => { Riffian; Latin; Netherlands }-->
+		<!--{ Riffian; ?; ? } => { Riffian; Latin; Morocco }-->
```

Comment:
@DavidLRowe I think the remainder of these likely tags are also implied.

Reply:
These replacement happens if the Tfng doesn't change the country, and NL doesn't change the script.